Zeeguu Reader/Pods/Zeeguu-API-iOS/ZeeguuAPI/ZeeguuAPI/Article.swift

```diff
@@ -26,19 +26,98 @@
 
 import UIKit
 
-public class Article: CustomStringConvertible {
+/// Adds support for comparing `Article` objects using the equals operator (`==`)
+///
+/// - parameter lhs: The left `Article` operand of the `==` operator (left hand side) <pre><b>lhs</b> == rhs</pre>
+/// - parameter rhs: The right `Article` operand of the `==` operator (right hand side) <pre>lhs == <b>rhs</b></pre>
+/// - returns: A `Bool` that states whether the two `Article` objects are equal
+public func ==(lhs: Article, rhs: Article) -> Bool {
+	return lhs.feed == rhs.feed && lhs.title == rhs.title && lhs.url == rhs.url && lhs.date == rhs.date && lhs.summary == rhs.summary
+}
+
+/// The `Article` class represents an article. It holds the source (`feed`), `title`, `url`, `date`, `summary` and more about the article.
+public class Article: CustomStringConvertible, Equatable {
+	
+	// MARK: Properties -
+	
+	/// The `Feed` from which this article was retrieved
 	public var feed: Feed
+	/// The title of this article
 	public var title: String
+	/// The url of this article
 	public var url: String
+	/// The publication date of this article
 	public var date: String
+	/// The summary of this article
 	public var summary: String
+	
+	private var imageURL: String?
+	private var image: UIImage?
 	private var contents: String?
+	private var difficulty: ArticleDifficulty?
+	
+	/// Whether the contents of this article have been retrieved yet
```

Comment:
This is a file from the cocoa pod 😊. Comments about the Zeeguu API should go in the Zeeguu API repository 😜.

But since we are in grammar nazi mode 😜, **the contents** of this article **have** been retrieved, so I'm correct 😜